### PR TITLE
Add isTablet to Constants

### PR DESCRIPTION
**Pull request by M-i-k-e-l**

### CODE REVIEW

File: src/helpers/Constants.js [hunk 1]
```diff
@@ -40,6 +40,8 @@ export let screenWidth = width;
 export let screenHeight = height;
 export let isSmallScreen = screenWidth <= 340;
 export let isShortScreen = screenHeight <= 600;
+export const ratio = screenWidth < screenHeight ? screenHeight / screenWidth : screenWidth / screenHeight;
+export const isTablet = Platform.isPad || (ratio < 1.6 && Math.max(screenWidth, screenHeight) >= 900);
```
Comment (Inbal-Tish) [inline src/helpers/Constants.js]: Where did you get the 1.6 ratio limit?

File: src/helpers/Constants.js [hunk 2]
```diff
@@ -40,6 +40,8 @@ export let screenWidth = width;
 export let screenHeight = height;
 export let isSmallScreen = screenWidth <= 340;
 export let isShortScreen = screenHeight <= 600;
+export const ratio = screenWidth < screenHeight ? screenHeight / screenWidth : screenWidth / screenHeight;
```
Comment (Inbal-Tish) [inline src/helpers/Constants.js]: Why are you exporting ratio?

File: src/helpers/Constants.js (hunk 1 above)
Comment (M-i-k-e-l) [reply src/helpers/Constants.js]: It was a guesstimation in a Stack-Overflow answer.

File: src/helpers/Constants.js (hunk 2 above)
Comment (M-i-k-e-l) [reply src/helpers/Constants.js]: I don't need it exported, but thought it might be useful.
Do you want me to remove that? - Or maybe change the name?

Comment (Inbal-Tish) [reply src/helpers/Constants.js]: Let's remove the export till we need it and change the name to be more descriptive.

File: src/helpers/Constants.js (hunk 1 above)
Comment (Inbal-Tish) [reply src/helpers/Constants.js]: Did you test it?

Comment (M-i-k-e-l) [reply src/helpers/Constants.js]: Yes, it worked on the devices I've tested; and I've added the screen size for good measure

Comment (Inbal-Tish) [reply src/helpers/Constants.js]: Cool

File: src/helpers/Constants.js (hunk 2 above)
Comment (M-i-k-e-l) [reply src/helpers/Constants.js]: Done

**Pull request merged**